renovate.json

```diff
@@ -10,6 +10,11 @@
         "matchBaseBranches": ["release-2.10","release-2.9"],
         "packagePatterns": ["*"],
         "enabled": false
+      },
+      {
+        "packagePatterns": ["*"],
+        "matchCurrentVersion": "/^v\\d+.\\d+.\\d+-(\\d+\\.)?\\d{14}-[a-f0-9]+/",
+        "enabled": false
```

Comment:
Can you please explain how this is supposed to work?

Reply:
For example, I think

```
	github.com/opentracing/opentracing-go v1.2.1-0.20220228012449-10b1cf09e00b
```

would match this as well, but it's not replaced.

Reply:
Hi @pstibrany, I didn't find any info regarding the replace, I can see 2 options here:

1. ignore the update when we specify the commit for whatever the reason we have. until we manually remove those specification. 

2. Another way is to just add the package to ignore list. 

I don't have preference regarding the 2, they both have pros/cons, the first one is as soon as we removed the commit specified, the update would resume by itself, but we will not update some of the dependency even they have new commits. the option 2 is we have to remove the ignore manually when we no longer doing replace. what do you think? If you have better solution please let me know.

Reply:
I can improve this by limit the rule only for the library that being replaced. but it requires a manual update everytime we add a new replace.

Reply:
I would suggest ignoring packages that we replace. Perhaps we could also add a lint check to verify if list of replaces is subset of ignored packages (there can be other ignored packages too).

Reply:
The updated PR ignored the package we replace if they are still using specific commit. when replace removed, the normal update would continue. i can remove the tag match, but i think it is fine to have it for the case when we want to continue update replacing lib with tags.